rag_sql_demo/app.py

```
@@ -0,0 +1,543 @@
+import os
+import streamlit as st
+import sqlite3
+import pandas as pd
```

Ruff (0.8.2)

`pandas` imported but unused

Remove unused import: `pandas`

(F401)